fog/view/server/src/server.rs

```diff
@@ -128,6 +127,16 @@ where
                     .bind_using_uri(fog_view_uri, logger.clone())
             }
             ClientListenUri::Store(ref fog_view_store_uri) => {
+                let fog_view_service = view_grpc::create_fog_view_store_api(FogViewService::new(
+                    enclave.clone(),
+                    Arc::new(recovery_db),
+                    db_poll_thread.get_shared_state(),
+                    client_authenticator,
+                    config.client_listen_uri.clone(),
+                    sharding_strategy,
+                    logger.clone(),
+                ));
+                log::debug!(logger, "Constructed View Store GRPC Service");
```

Comment:
:wrench: similar to above consider if/what value the debug log adds
```suggestion
```

Reply:
Same as above.

Reply:
Same as above.